internal/security/setup/option/optionHelpers.go

```diff
@@ -214,25 +214,48 @@ func markComplete(dirPath string) error {
 	return nil
 }
 
-func getXdgRuntimeDir() string {
+func getWorkDir() string {
 	if xdgRuntimeDir, ok := os.LookupEnv(envXdgRuntimeDir); ok {
-		return xdgRuntimeDir
+		return filepath.Join(xdgRuntimeDir, pkiInitBaseDir)
+	} else if setup.Configuration.SecretsSetup.WorkDir != "" {
+		return setup.Configuration.SecretsSetup.WorkDir
 	}
-	// if $XDG_RUNTIME_DIR is undefined, default to /tmp
-	return defaultXdgRuntimeDir
+
+	// if not specified in config and $XDG_RUNTIME_DIR is undefined, default to /tmp
+	return filepath.Join(defaultWorkDir, pkiInitBaseDir)
+}
+
+func getCertConfigDir() (string, error) {
+	if setup.Configuration.SecretsSetup.CertConfigDir != "" {
+		return setup.Configuration.SecretsSetup.CertConfigDir, nil
+	}
+
+	return "", errors.New("Directory for certificate configuration files not configured")
 }
 
-func getPkiCacheDirEnv() string {
+func getCacheDir() string {
 	pkiCacheDir := os.Getenv(envPkiCache)
```

Comment:
Change to check whether env exists or not as well: ie `os.LookupEnv(envPkiCache)`

Reply:
We probably don't need to make this configurable through env var. The config toml should suffice. Unless, you think otherwise, I'll only check for config toml. If not specified, will use default value